.functional-tests.env

```diff
@@ -28,5 +28,5 @@ ADDRESS_LOOKUP_API_URL=https://whitelodge-ai-api.census-gcp.onsdigital.uk
 COOKIE_SETTINGS_URL=#
 EQ_SUBMISSION_CONFIRMATION_BACKEND=log
 VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS=35
-SDS_API_BASE_URL=http://localhost:5003
+SDS_API_BASE_URL=http://sds:5003
```

Comment:
Although this is mainly only used by action, the comment above is appropriate here also.